libyul/AsmParser.h

```diff
@@ -138,10 +138,10 @@ class Parser: public langutil::ParserBase
 	Case parseCase();
 	ForLoop parseForLoop();
 	/// Parses a functional expression that has to push exactly one stack element
-	Expression parseExpression();
+	Expression parseExpression(bool _builtinArgument=false);
```

Comment:
`_builtinArgument`, `UnlimitedLiteralArgument`, `literalArgument`, `literalBuiltinArgument`... All of these refer to the same thing so this is getting confusing. Let's stick to a single term. I think that "unlimited literal argument" is a good one because it's less ambiguous than "literal argument" (which is the usual term for this) but still close to it.

Reply:
Also:
```suggestion
	Expression parseExpression(bool _builtinArgument = false);
```
Same below.